src/modules/esl-panel-group/core/esl-panel-group.ts

```diff
@@ -32,13 +32,14 @@ export class ESLPanelGroup extends ESLBaseElement {
   @attr({defaultValue: 'auto'}) public fallbackDuration: number | 'auto';
   /** List of comma-separated "modes" to disable collapse/expand animation (for both Group and Panel animations) */
   @attr() public noCollapse: string;
-
   /**
    * Define accordion behavior
    * `single` allows only one Panel to be open.
    * `multiple` allows any number of open Panels.
    * */
   @attr({defaultValue: 'single'}) public accordionGroup: string;
+  /** Action params for show/hide requests from the group to child panels */
+  @jsonAttr() public actionParams: PanelActionParams;
```

Comment:
This is an object to merge to the params for child panels that panel group sends on show/hide requests when mode changed or reset called. This param is not a part of tab or other defined modes behavior
Open questions: 
1. name is discussable... it could also be `transitionParams`, `resetParams`
2. should it have a `{noCollapse: true}` value by default to prevent animation when mode changed (tabs -> accordion, open -> accordion, etc)?

Reply:
Just in case: questions  are still opened even with the latest updates

Reply:
2 - it seems to me that it would be good to set this parameter by default